tendrl/monitoring_integration/alert/utils.py

```diff
@@ -221,7 +221,10 @@ def find_cluster_short_name(integration_id):
         cluster = NS.tendrl.objects.Cluster(
             integration_id=integration_id
         ).load()
-        return cluster.short_name
+        if cluster.short_name in [None, '']:
+            return cluster.integration_id
+        else:
+            return cluster.short_name
```

Comment:
Is it really required? short_name is populated with integration_id if short_name is not provided right?

Reply:
Ack. Check this https://github.com/Tendrl/commons/blob/master/tendrl/commons/flows/import_cluster/__init__.py#L59 While import we set `short_name` as `integration_id` if its not provided in request.